velox/exec/tests/TableScanTest.cpp

```diff
@@ -13,11 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "velox/exec/TableScan.h"
+#include <fmt/ranges.h>
 #include <folly/synchronization/Baton.h>
 #include <folly/synchronization/Latch.h>
 #include <atomic>
 #include <shared_mutex>
+
```

Comment:
nit for header file ordering:

```
#include <atomic>
#include <shared_mutex>

#include <fmt/ranges.h>
#include <folly/synchronization/Baton.h>
#include <folly/synchronization/EventCount.h>
#include <folly/synchronization/Latch.h>
```

Note that also fix `#include "folly/experimental/EventCount.h"` to `#include <folly/synchronization/EventCount.h>`

Reply:
Thanks for the reminder. There are other headers to be cleaned up. Will send a separate PR for that.

Reply:
Just FYI, I personally think the include order should be local to global:

Imagine you are writing a library that depends on some other library. You might do something like this:

```
#include <vector> //STL
#include <gtest/gtest.h> //ThirdParty 
#include "geometry.hpp" //In-house
 
#define PI 3.2 //As pr. Indiana Bill #246, 1897
 
TEST(TestGeometry, rotatingOrigoGivesOrigo) {
    std::vector<double> v(2,0);
    rotate(v, PI);
    EXPECT_EQ(0, v[0]) << "X got moved!";
    EXPECT_EQ(0, v[1]) << "Y got moved!";
}
```
and your geometry.hpp looks like this:

```
void rotate(std::vector<double>& v, double angle);
```
This all works out nicely until someone else wants to include geometry.hpp without including <vector> first, and get something like geometry.hpp:1: error: ‘vector’ is not a member of ‘std’. **You have now forced all the users of your geometry library to include <vector> before including geometry.hpp**.

But the maintainers think this is less an issue and no resource to update all include headers now.

Reply:
I have the similar preference as https://github.com/facebookincubator/velox/issues/1918#issuecomment-1172541369, as well as the Google code style for the header ordering https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.

I did not see much difference as your proposed style.